src/sentry/static/sentry/app/components/events/eventTags/eventTagsPill.tsx

```diff
@@ -73,6 +77,22 @@ const EventTagsPill = ({
           </VersionHoverCard>
         </div>
       )}
+      {isTrace && (
+        <TraceHoverCard
+          containerClassName="pill-icon"
+          traceId={tag.value}
+          orgId={orgId}
+          location={location}
+        >
+          {({to}) => {
+            return (
+              <Link to={to}>
```

Comment:
Should this link be shown to users without discover2? They could be sent to a page that has an upsell state. In other parts of the application we've been showing disabled buttons/links when the user doesn't have access to discover.

Reply:
good call, changed to not show the icon if the user does not have discover